app/src/main/java/emu/skyline/input/onscreen/OnScreenControllerView.kt

```diff
@@ -258,6 +258,32 @@ class OnScreenControllerView @JvmOverloads constructor(context : Context, attrs
         invalidate()
     }
 
+    fun increaseOpacity() {
+        for (button in controls.allButtons) {
+            var currentOpactity = button.config.opacity
+            currentOpactity += 10
+            if (currentOpactity > 100) currentOpactity = 100
+            button.config.opacity = currentOpactity
+            if (button is JoystickButton) {
+                button.innerButton.config.opacity = currentOpactity
+            }
+        }
+        invalidate()
+    }
+
+    fun decreaseOpacity() {
```

Comment:
I feel like having two separate functions to do almost the same thing is kind of redundant. You could instead have one function that takes the value to add or remove as parameter, then clamp the value to 0-100 range.

Reply:
That's what i done, maybe we should do the same with increase and decrease globalScale fucntiones